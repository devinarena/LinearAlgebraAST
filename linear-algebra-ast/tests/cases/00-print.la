print 0.0;
print 3.1415962;
print 5.5;