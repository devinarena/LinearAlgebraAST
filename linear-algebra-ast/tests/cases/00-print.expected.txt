TOKEN_PRINT - print - 1
TOKEN_NUMBER - 0.0 - 1
TOKEN_SEMICOLON - ; - 1
TOKEN_PRINT - print - 2
TOKEN_NUMBER - 3.1415962 - 2
TOKEN_SEMICOLON - ; - 2
TOKEN_PRINT - print - 3
TOKEN_NUMBER - 5.5 - 3
TOKEN_SEMICOLON - ; - 3
TOKEN_EOF -  - 3
0
3.1415962
5.5
